### PR TITLE
Revise Princeton mapping, fixes #354

### LINKED ISSUE #354: Check, revise, and transform Princeton
After significant changes to the IR, it is necessary to check, possibly revise, re-transform, and re-load the Princeton data sets.

### PR DESCRIPTION
**Pull request by jacobthill**
## Why was this change made?

- Revise Princeton mapping

## Was the documentation (README, API, wiki, ...) updated?

- n/a

### CODE REVIEW

**Comment by jacobthill** [comment]
@aaron-collier, @mjgiarlo This error seems to be related to the start-date being later than the end-date which I thought was fixed. Any idea how to resolve this?

File: traject_configs/princeton_yemeni_mss_config.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Cho Other
```
Comment (mjgiarlo) [inline traject_configs/princeton_yemeni_mss_config.rb]: @jacobthill I believe one of the errors in this branch may be due to using the `lang()` macro without having required/extended the DLME macros module in this config. cc: @aaron-collier

Comment (jacobthill) [reply traject_configs/princeton_yemeni_mss_config.rb]: @mjgiarlo that didn't seem to work. Also this is a typical pattern. Whenever I have a common config with collection specific configs as well, all the requirements/enxtensions are in the common but not repeated in the collection specific configs (e.g. bnf, michigan, etc.). Also this seems to be complaining about a michigan record though nothing in this PR should effect michigan (until I just added the extensions). Am I missing something?

Comment (jacobthill) [reply traject_configs/princeton_yemeni_mss_config.rb]: Nevermind, I figured it out. There were two problems (with michigan and penn_marc).

Comment (mjgiarlo) [reply traject_configs/princeton_yemeni_mss_config.rb]: Good find!

**Pull request merged**